conans/client/tools/oss.py

```diff
@@ -185,8 +185,13 @@ def with_apt(self):
     @property
     def with_yum(self):
         return self.is_linux and self.linux_distro in \
-                                 ("centos", "redhat", "fedora", "pidora", "scientific",
-                                  "xenserver", "amazon", "oracle", "rhel")
+                                 ( "pidora", "scientific",
+                                  "xenserver", "amazon", "oracle")
+
+    @property
+    def with_dnf(self):
```

Comment:
what about older Fedora versions? I think you can check DNF first, but need to keep YUM as an option.

Reply:
So go to `with_yum` as a fallback when using `with_dnf`?

`dnf` was added in Fedora 18 (2013) and default in Fedora 22 (2015), so depends on how old this needs to be backwards compatible?

Reply:
ow, 4 years ago. So I think is good keeping as default.

Reply:
Thinking again, it will be better keeping as fallback. Centos 6 and 7 don't have dnf intalled by default, and Orgs like Bincrafters are using CentOS6 to build installers due glibc version.

Reply:
Yes, definitely better to keep the same behavior for fedora and maybe we can detect if that dnf is there or not.

Reply:
`tools.which("dnf")`

Reply:
Please @ShadowMitia check that `tools.witch("dnf") is not None` to use the `with_dnf` or not